fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/domain/SavingsAccount.java

```diff
@@ -140,11 +140,11 @@
     @JoinColumn(name = "client_id", nullable = true)
     protected Client client;
 
-    @ManyToOne(optional = true)
+    @ManyToOne(optional = true, fetch=FetchType.LAZY)
     @JoinColumn(name = "group_id", nullable = true)
     protected Group group;
 
-    @ManyToOne
+    @ManyToOne(fetch=FetchType.LAZY)
```

Comment:
```suggestion
    @ManyToOne(fetch=FetchType.LAZY) // see FINERACT-902
```

Reply:
@rrpawar96 did you want to add this?

Reply:
Yes @vorburger, I want to add this annotation and I will implement the fetchplan() in the future enhancement.